profiles/serializers.py

```diff
@@ -69,16 +88,34 @@ def get_pretty_printed_student_id(self, obj):
     def update(self, instance, validated_data):
         with transaction.atomic():
             for attr, value in validated_data.items():
-                if attr == 'work_history':
+                if attr == 'work_history' or attr == 'education':
                     continue
                 else:
                     setattr(instance, attr, value)
-
             instance.save()
             if 'work_history' in validated_data:
                 update_work_history(validated_data['work_history'], instance.id)
 
-        return instance
+            profile_id = instance.id
```

Comment:
It would be nice to move this block of code into an `update_education` function similar to `update_work_history` above, but we can do this in another PR